src/main/resources/assets/js/components/AppComponent.jsx

```diff
@@ -20,6 +23,45 @@ var AppComponent = React.createClass({
       "text-warning": isDeploying
     });
 
+    var healthData = [
```

Comment:
Let's get all this health data logic into the App.js model. This is not presenter logic.